docs/installation.md

```diff
@@ -33,15 +33,16 @@ apimock.processor.process({
     src: 'mocks', // required
     patterns: { // optional
         mocks: '**/*Mock.json', // optional: default is '**/*.mock.json'
-        presets: '**/*Preset.json' // optional: default is '**/*.preset.json'
+        presets: '**/*Preset.json', // optional: default is '**/*.preset.json'
+        mockWatches: '**/*.json' // optional: no default, set if watch files regex is different from mocks pattern
```

Comment:
@katienuccio can you change the parameter to from mockWatches to watches?
Then we can use it too for presets.

```
{ 
    src: 'mocks', 
    patterns: { 
        mocks: '**/*Mock.json',
        presets: '**/*Preset.json' 
    },
    watches: {
        mocks: '**/*.json',
        presets: '**/*'
    }
}
```

Reply:
fixed!